provision/ubuntu/kernel-next.sh

```diff
@@ -97,6 +97,11 @@ yes "" | make localyesconfig && make prepare
 ./scripts/config --module CONFIG_VXLAN
 ./scripts/config --module CONFIG_GENEVE
 ./scripts/config --module CONFIG_NET_SCH_FQ
+# Required for kernel SCTP Support.
```

Comment:
Can we change this to `Required for Cilium's SCTP tests` and move it after the IPsec configs, to clarify that it's only needed for tests and not by Cilium itself?

Reply:
Done.